src/reducers/ui-state-updaters.js

```diff
@@ -441,10 +431,35 @@ export const cleanupExportImageUpdater = state => ({
     ...state.exportImage,
     exporting: false,
     imageDataUri: '',
-    error: false
+    error: false,
+    center: false
   }
 });
 
+/**
+ * Start image exporting flow
+ * @memberof uiStateUpdaters
+ * @param state
+ * @param options
+ * @returns {UiState}
+ * @type {typeof import('./ui-state-updaters').startExportingImage}
+ * @public
+ */
+export const startExportingImageUpdater = (state, {payload: options = {}}) => {
+  const imageSettings = {
+    ...options,
+    exporting: true
+  };
+
+  const modifiers = [
```

Comment:
the `composet-helpers` functions are designed to compose updater functions with a simplier style
you can just do:
`import {payload_, apply_, compose_} from './composer-helpers';`
then
```js
  return compose_([
    cleanupExportImageUpdater,
    apply_(setExportImageSettingUpdater, payload_(imageSettings)),
    apply_(toggleModalUpdater, payload_(EXPORT_IMAGE_ID)),
  ])(state);
```